_site/index.html

```diff
@@ -0,0 +1,186 @@
+<head>
```

Comment:
Plotly is used to display graphs ✅

We will make the dashboard much more dynamic and interactive I guess ? @marcin-krystianc

Reply:
Yes, we can also use any other library if you find it useful. The only limitation here is that we don't want to have any server-side components. Everything should be client-side only so we can keep hosting it on gh-pages.